ide_profiles/xcode/ExtendedKF.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,680 @@
+				FF109B1E5298466D82B9A2D8 /* /Users/denyskrut/Documents/Extended-Kalman-Filter/src/main.cpp */,
+				419A3278AFC64F669774B9CA /* /Users/denyskrut/Documents/Extended-Kalman-Filter/src/tools.cpp */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXSourcesBuildPhase section */
+
+/* Begin PBXTargetDependency section */
+		250204BDAF9B4ED2B6AD1520 /* PBXTargetDependency */ = {
+			isa = PBXTargetDependency;
+			target = D68CA58B14FB4101AC2B934E /* ExtendedKF */;
+			targetProxy = 1944C55DCFBF43799917DD75 /* PBXContainerItemProxy */;
+		};
+		AED8632805814288A9DDF4C7 /* PBXTargetDependency */ = {
+			isa = PBXTargetDependency;
+			target = 310FF99FFD1540B0A274F1C0 /* ZERO_CHECK */;
+			targetProxy = 186457E584D546318CD0CA64 /* PBXContainerItemProxy */;
+		};
+		E0E9A5AE00404E9DAF7225F4 /* PBXTargetDependency */ = {
+			isa = PBXTargetDependency;
+			target = 310FF99FFD1540B0A274F1C0 /* ZERO_CHECK */;
+			targetProxy = F6520AA0FB3D481FBE750EEA /* PBXContainerItemProxy */;
+		};
+/* End PBXTargetDependency section */
+
+/* Begin XCBuildConfiguration section */
+		0653465B620F4E63B0577424 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				COMBINE_HIDPI_IMAGES = YES;
+				GCC_GENERATE_DEBUGGING_SYMBOLS = YES;
+				GCC_INLINES_ARE_PRIVATE_EXTERN = NO;
+				GCC_OPTIMIZATION_LEVEL = 0;
+				GCC_PREPROCESSOR_DEFINITIONS = ("'CMAKE_INTDIR=\"$(CONFIGURATION)$(EFFECTIVE_PLATFORM_NAME)\"'");
+				GCC_SYMBOLS_PRIVATE_EXTERN = NO;
+				INSTALL_PATH = "";
+				OTHER_LDFLAGS = " ";
+				OTHER_REZFLAGS = "";
+				PRODUCT_NAME = ALL_BUILD;
+				SECTORDER_FLAGS = "";
+				SYMROOT = "/Users/denyskrut/Documents/Extended-Kalman-Filter/ide_profiles/xcode";
+				USE_HEADERMAP = NO;
+				WARNING_CFLAGS = ("-Wmost","-Wno-four-char-constants","-Wno-unknown-pragmas","$(inherited)");
+			};
+			name = Debug;
+		};
+		08E03646CD8A4446BAA75A5B /* MinSizeRel */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				COMBINE_HIDPI_IMAGES = YES;
+				GCC_GENERATE_DEBUGGING_SYMBOLS = YES;
+				GCC_INLINES_ARE_PRIVATE_EXTERN = NO;
+				GCC_OPTIMIZATION_LEVEL = 0;
+				GCC_PREPROCESSOR_DEFINITIONS = ("'CMAKE_INTDIR=\"$(CONFIGURATION)$(EFFECTIVE_PLATFORM_NAME)\"'");
+				GCC_SYMBOLS_PRIVATE_EXTERN = NO;
+				INSTALL_PATH = "";
+				OTHER_LDFLAGS = " ";
+				OTHER_REZFLAGS = "";
+				PRODUCT_NAME = ZERO_CHECK;
+				SECTORDER_FLAGS = "";
+				SYMROOT = "/Users/denyskrut/Documents/Extended-Kalman-Filter/ide_profiles/xcode";
+				USE_HEADERMAP = NO;
+				WARNING_CFLAGS = ("-Wmost","-Wno-four-char-constants","-Wno-unknown-pragmas","$(inherited)");
+			};
+			name = MinSizeRel;
+		};
+		2A332E90F4464DF098D32B63 /* MinSizeRel */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ONLY_ACTIVE_ARCH = YES;
+				SDKROOT = /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk;
+				SYMROOT = "/Users/denyskrut/Documents/Extended-Kalman-Filter/ide_profiles/xcode/build";
```

Comment:
it looks like this path (and a few others) are specific to your machine. is there a way around this? could students generate this file themselves?